components/KitButtonSupreme.js

```diff
@@ -47,7 +49,11 @@ export default function KitButtonSupreme({
   });
   return (
     <View style={style}>
-      <TouchableOpacity style={styles.button} onPress={onPress}>
+      <TouchableOpacity
+        style={styles.button}
+        onPress={onPress}
+        disabled={disabled}
```

Comment:
Is there a UI change when a button is disabled?

Reply:
hm currently it's up to the caller

Reply:
hm no, currently it's up to the caller but it would make sense to make it part of KitButtonSupreme